modules/shared/components/molecules/Footer/Footer.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import type { FC, ReactElement } from 'react';
+import type { IFooter } from './IFooter';
+import Share from '../../icons/share.svg';
+import styles from './Footer.module.css';
+
+const Footer: FC<IFooter.IProps> = ({
+  numberOfVotes,
+  showResult = false,
```

Comment:
why do you need this flag here? isn't it enough to derive it from the `!numberOfVotes`?